packages/node_modules/@webex/plugin-meetings/src/stats/data.js

```diff
@@ -13,6 +16,28 @@ export default class WebRTCData {
     this.id = uuid.v4();
   }
 
+  /**
+   * get flattened rtc/rtcp/rtp/track/transport/candidate data with omitted data (default)
+   * @returns {Object}
+   */
+  flatten() {
+    let flat = {};
+
+    Object.keys(this.data).forEach((key) => {
```

Comment:
Nit: I think `reduce` could accomplish the same thing without rewriting the flat variable in place.